plugin/forward/forward.go

```diff
@@ -206,4 +218,27 @@ const (
 	sequentialPolicy
 )
 
+type protoFlags int
```

Comment:
why not just a  struct with the options as boolean members?

Reply:
so, would you prefer to have the struct like
```
type options struct {
  forceTcp bool
  preferUdp bool
  metrics bool // do we need this?
}
```